scripts/create_release_pr.sh

```diff
@@ -36,13 +36,11 @@
 #   |   v2.0.8-rc.4   |      -z       |       v2.0.8      |
 #   |     v2.0.8      |      -n       |     v2.0.9-rc.1   |
 #   |   v2.0.8-rc.4   |      -n       |     v2.0.8-rc.5   |
-#   |   v2.0.8-rc.4   | -v 4.1.2-rc.7 |     v4.1.2-rc.7   |
```

Comment:
I was using this exact versions few times, when things were needed to be fixed or to just produce versions which I could store in my private quay registry, how to do this now?

Reply:
I really want to remove this option, if we have written everything correctly, we do not need that option